CppCoreGuidelines.md

```diff
@@ -19031,6 +19032,28 @@ Failing to follow this results in difficult to diagnose errors for clients of a
 
 A test should verify that the header file itself compiles or that a cpp file which only includes the header file compiles.
 
+### <a name="Rs-incscope"></a>SF.12: Use the quoted or angle bracket form of `#include`  to identify the scope of dependencies
```

Comment:
Two spaces before "to"? Maybe it is rendered as one space, but wouldn't it be better to only write one space here?